std/meta.d

```diff
@@ -249,19 +249,20 @@ if (!isAggregateType!T || is(Unqual!T == T))
 }
 
 /**
- * Returns the index of the first occurrence of T in the
- * sequence TList.
- * If not found, -1 is returned.
+ * Returns the index of the first occurrence of `args[0]` in the
+ * sequence `args[1 .. $]`. `args` may be types or compile-time values.
+ * If not found, `-1` is returned.
  */
-template staticIndexOf(T, TList...)
+template staticIndexOf(args...) if (args.length >= 1)
```

Comment:
The `if` needs to be moved to the next line.

Reply:
thanks!